core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -47,7 +47,8 @@
 		adr	x2, stack_tmp_stride
 		ldr	w1, [x2]
 		mul	x2, x0, x1
-		adr	x1, stack_tmp
+		adrp	x1, stack_tmp
+		add	x1, x1, :lo12:stack_tmp
```

Comment:
`ldr x1, =stack_tmp` instead? Would be more legible, and it's also how it's done in `generic_entry_a32.S`.
(Same for `__init_end` and `__end` below, obviously)